lib/matplotlib/colors.py

```diff
@@ -1566,17 +1587,23 @@ def autoscale_None(self, A):
             in_trf_domain = np.extract(np.isfinite(self._trf.transform(A)), A)
             return super().autoscale_None(in_trf_domain)
 
-    Norm.__name__ = (f"{scale_cls.__name__}Norm" if base_norm_cls is Normalize
-                     else base_norm_cls.__name__)
-    Norm.__qualname__ = base_norm_cls.__qualname__
+    Norm.__name__ = (
+            f"{scale_cls.__name__}Norm" if base_norm_cls is Normalize
+            else base_norm_cls.__name__)
+    Norm.__qualname__ = (
+            f"{scale_cls.__qualname__}Norm" if base_norm_cls is Normalize
+            else base_norm_cls.__qualname__)
     Norm.__module__ = base_norm_cls.__module__
     Norm.__doc__ = base_norm_cls.__doc__
-    Norm.__init__.__signature__ = bound_init_signature.replace(parameters=[
-        inspect.Parameter("self", inspect.Parameter.POSITIONAL_OR_KEYWORD),
-        *bound_init_signature.parameters.values()])
+
     return Norm
 
 
+def _picklable_norm_constructor(*args, **kwargs):
+    cls = _make_norm_from_scale(*args, **kwargs)
+    return cls.__new__(cls)
```

Comment:
Is that right? `_make_norm_from_scale` doesn't take any keyword arguments, and I'm not sure the `__reduce__` callable needs to take any.

Reply:
Sure, I dropped kwargs, I guess it doesn't really matter?

Reply:
Well, I was confused at first, and thought `self.__dict__` was becoming `kwargs`, but then I read the docs for `__reduce__`., so it seems better to leave out an unused `kwargs`.